docs/reference/pixi_configuration.md

```diff
@@ -0,0 +1,209 @@
+    | 2            | `%APPDATA%\pixi\config.toml`                                           | User-specific configuration                                                                    |
+    | 3            | `$PIXI_HOME\config.toml`                                               | Global configuration in the user home directory. `PIXI_HOME` defaults to `%USERPROFILE%/.pixi` |
+    | 4            | `your_project\.pixi\config.toml`                                       | Project-specific configuration                                                                 |
+    | 5            | Command line arguments (`--tls-no-verify`, `--change-ps1=false`, etc.) | Configuration via command line arguments                                                       |
+
+!!! note
+    The highest priority wins. If a configuration file is found in a higher priority location, the lower priority locations are overwritten.
+
+
+!!! note
+    To find the locations where `pixi` looks for configuration files, run
+    `pixi` with `-v` or `--verbose`.
+
+## Reference
+
+??? info "Casing In Configuration"
+    In versions of pixi `0.20.1` and older the global configuration used snake_case
+    we've changed to `kebab-case` for consistency with the rest of the configuration.
+    But we still support the old `snake_case` configuration, for older configuration options.
+    These are:
+
+    - `default_channels`
+    - `change_ps1`
+    - `tls_no_verify`
+    - `authentication_override_file`
+    - `mirrors` and sub-options
+    - `repodata-config` and sub-options
+
+The following reference describes all available configuration options.
+
+### `default-channels`
+
+The default channels to select when running `pixi init` or `pixi global install`.
+This defaults to only conda-forge.
+```toml title="config.toml"
+default-channels = ["conda-forge"]
+```
+
+### `change-ps1`
+
+When set to false, the `(pixi)` prefix in the shell prompt is removed.
+This applies to the `pixi shell` subcommand.
+You can override this from the CLI with `--change-ps1`.
+
+```toml title="config.toml"
+change-ps1 = true
+```
+
+### `tls-no-verify`
+When set to true, the TLS certificates are not verified.
+
+!!! warning
+
+    This is a security risk and should only be used for testing purposes or internal networks.
+
+You can override this from the CLI with `--tls-no-verify`.
+
+```toml title="config.toml"
+tls-no-verify = false
+```
+
+### `authentication-override-file`
+Override from where the authentication information is loaded.
+Usually, we try to use the keyring to load authentication data from, and only use a JSON
+file as a fallback. This option allows you to force the use of a JSON file.
+Read more in the authentication section.
+```toml title="config.toml"
+authentication-override-file = "/path/to/your/override.json"
+```
+
+### `target-environment-directory`
```

Comment:
Will this change also affect pixi global environments?

In my setup, all `HOME` is persisted on a network drive. When this affects the global installation tools as well, this could break some setups

Reply:
No it doesn't indeed because of the `PIXI_HOME` setting